### PR TITLE
Small changes to codeowners and contribution policy

### PR DESCRIPTION
**Pull request by asaj**
### Description

Clarify code owners and PR review policy to reduce the barrier to entry for contributions.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,15 +2,31 @@
...
+
 # directory and file-level owners. Feel free to add to this!
 /packages/analytics/ @nambrot @timmoreton
+/packages/attestation-service/ @nambrot
+/packages/azure/ @codyborn
 /packages/blockchain-api/ @jmrossy @cmcewen
-/packages/cli/ @mcortesi @asaj
+/packages/celotool/ @asaj
+/packages/cli/ @mcortesi
+/packages/contractkit/ @mcortesi
```
Comment (jmrossy) [inline .github/CODEOWNERS]: '+ @gastonponti ?

**Review by jmrossy — APPROVED** [review]
LGTM, one optional comment below

**Review by timmoreton — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -2,15 +2,31 @@
...
+/packages/dappkit/ @nambrot
```
Comment (nambrot) [inline .github/CODEOWNERS:16]: I'd say at this point I'm so far removed that I think it makes sense for someone on the mobile/sdk side to take this?

**Review by nategraf — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (gastonponti) [reply .github/CODEOWNERS]: I could be added to the `contractkit` and `cli` if you want. There are a few areas of those that I never used before, but I can totally help @mcortesi with that

Comment (mcortesi) [reply .github/CODEOWNERS]: love it :D

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -2,15 +2,31 @@
...
+/packages/devutils/ @mcortesi
+/packages/docs/ @timmoreton
+/packages/faucet/ @mcortesi
```
Comment (mcortesi) [inline .github/CODEOWNERS]: very much oudated with this.. maybe @aaronmgdr ?

Comment (aaronmgdr) [reply .github/CODEOWNERS]: I think @jmrossy is the one who really knows it. I dont know if ive ever touched it

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -2,15 +2,31 @@
...
 /packages/helm-charts/ @nambrot @timmoreton
+/packages/leaderboard/ @mrsmkl
+/packages/metadata-crawler/ @aaitor
 /packages/mobile/ @cmcewen @jmrossy
```
Comment (aaronmgdr) [inline .github/CODEOWNERS:23]: should @i1skn  or @jeanregisser be here?

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -2,15 +2,31 @@
...
+/packages/moonpay-auth/ @annakaz
 /packages/notification-service/ @annakaz @cmcewen
-/packages/protocol/ @asaj @m-chrzan
+/packages/phone-number-privacy/ @jmrossy
+/packages/protocol/ @asaj @nategraf
 /packages/react-components/ @cmcewen @jmrossy
```
Comment (aaronmgdr) [inline .github/CODEOWNERS]: I might make sense here on react-components

File: .github/CODEOWNERS (hunk 4 above)
Comment (asaj) [reply .github/CODEOWNERS:23]: You tell me :)

File: .github/CODEOWNERS (hunk 2 above)
Comment (asaj) [reply .github/CODEOWNERS:16]: Any suggestions?

**Pull request merged**